### ISSUE TITLE
wx.adv.EVT_NOTIFICATION_MESSAGE_* Events Not implemented?

### ISSUE THREAD

**Issue by j4c3** [wxWidgets/Phoenix#1986]
The api docs reference notification events here: https://docs.wxpython.org/wx.adv.NotificationMessage.html

But, they do not appear to be defined (4.1.1, Windows 10, in VSCode.)  

I had posted to the forums, but it was suggested to mention it here.

**Comment by swt2c** [comment]
Yes, valid bug.  They seem to be missing from the interface headers.  Opened up a PR to fix it upstream: https://github.com/wxWidgets/wxWidgets/pull/2475

**Comment by j4c3** [comment]
These appear to have been added upstream, does this need to have them added in notifmsg.py as below as well?

module.addPyCode("""\
        EVT_NOTIFICATION_MESSAGE_CLICK = wx.PyEventBinder (         wxEVT_NOTIFICATION_MESSAGE_CLICK )
        EVT_NOTIFICATION_MESSAGE_DISMISSED = wx.PyEventBinder (    wxEVT_NOTIFICATION_MESSAGE_DISMISSED )
        EVT_NOTIFICATION_MESSAGE_ACTION = wx.PyEventBinder (      wxEVT_NOTIFICATION_MESSAGE_ACTION )
        """)

**Comment by swt2c** [comment]
Yes, thanks for the reminder.  However the wxWidgets reference needs to be updated as well.  I'm hesitant to do that without Robin's okay, though, as I don't know what his plans are.

**Comment by j4c3** [comment]
I just checked on this in 4.2, and it appears to remain un-implemented.
